src/main/java/sirius/biz/model/PermissionData.java

```diff
@@ -41,14 +41,14 @@
     private final BaseEntity<?> parent;
 
     /**
-     * Contains all permissions as a single string, separated with commas.
+     * Contains all permissions.
      */
-    public static final Mapping PERMISSION_STRING = Mapping.named("permissionString");
+    public static final Mapping PERMISSIONS = Mapping.named("permissions");
     @Autoloaded
     @NullAllowed
     @AutoImport
     @Length(4096)
```

Comment:
Does this still need to be here?

Reply:
yes, so the Column in an SQL DB knows how long it should be, because in MySQL it is still stored as an string